.github/workflows/validate-workflow.yml

```
@@ -16,7 +16,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [14.x, 16.x, 18.x]
+        node-version: [18.16]
```

this should be an array of node versions that we have tested and currently support. Ideally, this should be based off of https://nodejs.dev/en/about/releases/... for now, we might want to consider extending support to 19 and 20.

```suggestion
        node-version: [14.x, 16.x, 18.x, 19.x]
```